traffic_monitor_golang/traffic_monitor/config/config.go

```diff
@@ -21,7 +21,6 @@ package config
 
```

Comment:
The config changes have been moved into https://github.com/apache/incubator-trafficcontrol/pull/620 which has been merged. I'll rebase this to remove them and resolve the conflicts.